### ISSUE TITLE
Exercises already filled out

### ISSUE THREAD

**Issue by OkilyDokily** [orlandpm/Math-for-Programmers#21]
It seems like the notebooks should serve two purposes. 1. to provide code boilerplate code and be able to follow along and play with the code that has been created in the book. And 2, let the user do the exercises in the context of the existing boilerplate.

For some reason all of the exercises are already solved which takes away much of the advantage of using the notebook. Why aren't there two notebooks? One for following along with the book and solving exercises, and another for seeing the solutions to those exercises?

There is a hidden folder with a copy of the notebook but which have the same name except append the file name with "-checkpoint" for some unexplained reason.

**Comment by ttots-belak** [comment]
I agree, it's frustrating enough having the solutions right next to the questions.
